src/app/views/loan-detail/transaction-history/transaction-history.component.scss

```diff
@@ -1,4 +1,9 @@
 @import '../../../../scss/_variables.scss'; //variables
+      
+.overlay-setup{
```

Comment:
Try centering with flex-box or 
  position: absolute;
  left: 50%;
  @include transform(translate(-50%, 0%));

Because the box its not realy centered with only left 50%